ehr/resources/queries/study/treatment_order.query.xml

```diff
@@ -60,7 +60,7 @@
                         <columnTitle>Frequency</columnTitle>
                         <fk>
                             <fkDbSchema>ehr_lookups</fkDbSchema>
-                            <fkTable>treatment_frequency</fkTable>
```

Comment:
Comment: in general I dont like to encode the FK using a target that is filtered. This will be that legacy data now has broken FKs. In the situation where the desired behavior is to exclude inactive treatment_frequency records, I typically do one of:

1) in the data entry metadata, add this filter. this would mean that any EHR-related entry code would limit new records to the active frequencies only

2) in some cases you can add "<fkDisplayColumnName useRawValue="true"/>", which would at least mean the data region would render a reasonable value. i forget what the raw values of treatment_frequency are, but if they're strings maybe that's viable here.

Reply:
Good point, change is now reverted with an addition of a filter which will get applied only during insert and update, and not when viewing the data.

Reply:
When did this LK feature get added to support <filters> in the query.xml?   That's useful....

Reply:
Looks like it got added early last year